index.html

```diff
@@ -3,10 +3,19 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- go get imports -->
     <meta name="go-import" content="gonum.org/v1/gonum git https://github.com/gonum/gonum">
     <meta name="go-import" content="gonum.org/v1/plot git https://github.com/gonum/plot">
     <meta name="go-import" content="gonum.org/v1/netlib git https://github.com/gonum/netlib">
     <meta name="go-import" content="gonum.org/v1/hdf5 git https://github.com/gonum/hdf5">
+
+    <!-- redirection to godoc documentation -->
+    <meta name="go-source" content="gonum.org/v1/gonum https://github.com/gonum/gonum/ https://github.com/gonum/gonum/tree/master{/dir} https://github.com/gonum/gonum/blob/master{/dir}/{file}#{line}"/>
```

Comment:
Two of these have an L in between {file} and {line}, the other two do not. Is that intentional?